dist/index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <!-- Fonts -->
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Ubuntu&display=swap" rel="stylesheet">
+  <title>Leaderboard</title>
+<script defer src="runtime.bundle.js"></script><script defer src="index.bundle.js"></script></head>
+
+<body>
+  <h1 class="title">Leaderboard</h1>
+
+  <div class="whole-container">
+    <!-- Score List -->
+    <div class="score-list">
+      <div class="score-header">
+        <h2>Recent Scores</h2>
+        <button class="refresh-btn btn">Refresh</button>
+      </div>
+      <ul class="scores-list" id="scores">
+        <!-- Scores -->
+      </ul>
+    </div>
+
+    <!-- Add your Score -->
+    <div class="add-score-container">
+      <div class="input-header">
+        <h2 class="input-title">Add your score</h2>
+      </div>
+      <div>
+        <form class="input-form">
+          <input id="name-input" type="text" name="name" placeholder="Your name" required>
+          <input id="score-input" type="text" name="score" placeholder="Your Score" required>
```

Comment:
- [ ] [OPTIONAL] 😉 Kindly make the `type` of this input `number` for semantic purposes. 😃